### PR TITLE
fix(DTFS2-6047): GEF validation issue facility 

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Issue:
* Issuing facility page went to error page if special characters in date boxes

# Changes made:
* Added validation on issue facility page if errors 
* Added helper
* Added tests

### CODE REVIEW

File: gef-ui/server/controllers/unissued-facilities/validation.test.js [hunk 1]
```diff
@@ -345,6 +345,296 @@ describe('validation()', () => {
     expect(result.aboutFacilityErrors).toEqual(expectedFacilityErrors);
   });
 
+  it('should return object with errors populated if issue date has errors', async () => {
```
Comment (abhi-markan) [inline gef-ui/server/controllers/unissued-facilities/validation.test.js:348]: Can abstract the test cases to more illustrative blocks i.e. 
* should return error when date is blank...
* Should return error when date has symbol..

Can we please add a test case for either date/month/year to be blank please.

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Good work, just one minor comment to address please.

File: gef-ui/server/controllers/unissued-facilities/validation.test.js (hunk 1 above)
Comment (Zainzzkk) [reply gef-ui/server/controllers/unissued-facilities/validation.test.js]: done

**Review by abhi-markan — APPROVED** [review]
LGTM

**Pull request merged**